packages/myst-parser/src/myst.ts

```diff
@@ -59,7 +60,17 @@ function parseOptions(opts?: Options): AllOptions {
 export function createTokenizer(opts?: Options) {
   const parsedOpts = parseOptions(opts);
   const { extensions, markdownit } = parsedOpts;
-  const tokenizer = MarkdownIt(MARKDOWN_IT_CONFIG as any, markdownit);
+  const tokenizer = MarkdownIt(
+    {
+      ...MARKDOWN_IT_CONFIG,
+      options: {
+        ...MARKDOWN_IT_CONFIG.options,
+        typographer: extensions.smartquotes ? true : false,
+      },
+    } as any,
+    markdownit,
+  );
+  if (extensions.smartquotes) tokenizer.enable('smartquotes');
```

Comment:
Then, once `typographer` is `true`, enabling `smartquotes` is still required to actually add the feature: https://github.com/markdown-it/markdown-it/blob/e476f78bc3ea3576beb61bdc94322d0a6b2d85cc/lib/parser_core.js#L19